data_steward/cdr_cleaner/cleaning_rules/remove_row_duplicates.py

```diff
@@ -0,0 +1,136 @@
+"""
+All columns except ID column in each domain should be unique
+"""
+import logging
+
+# Project imports
+import cdm
+from resources import fields_for
+from constants.cdr_cleaner import clean_cdr as cdr_consts
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule, query_spec_list
+from cdr_cleaner.clean_cdr_utils import get_tables_in_dataset
+from common import JINJA_ENV
+
+LOGGER = logging.getLogger(__name__)
+
+JIRA_ISSUE_NUMBERS = ['DC1732']
+
+DE_DUP_SANDBOX_QUERY_TEMPLATE = JINJA_ENV.from_string("""
+CREATE OR REPLACE TABLE `{{project_id}}.{{sandbox_dataset_id}}.{{sandbox_table_name}}` AS
+SELECT
+    t.* EXCEPT (row_num)
+FROM (
+    SELECT 
+        *,
+        ROW_NUMBER() OVER (PARTITION BY {{cols}}) AS row_num
+    FROM `{{project_id}}.{{dataset_id}}.{{table_name}}`
+) AS t
+WHERE row_num > 1
+""")
+
+DE_DUP_QUERY_TEMPLATE = JINJA_ENV.from_string("""
+DELETE
+FROM `{{project_id}}.{{dataset_id}}.{{table_name}}`
+WHERE {{table_name}}_id IN
+(SELECT {{table_name}}_id
+FROM `{{project_id}}.{{sandbox_dataset_id}}.{{sandbox_table_name}}`)
+""")
+
+
+class DeduplicateExceptIdColumn(BaseCleaningRule):
+
+    def __init__(self, project_id, dataset_id, sandbox_dataset_id):
+        """
+        Initialize the class with proper information.
+
+        Set the issue numbers, description and affected datasets. As other tickets may affect
+        this SQL, append them to the list of Jira Issues.
+        DO NOT REMOVE ORIGINAL JIRA ISSUE NUMBERS!
+        """
+        desc = (
+            'Remove the duplicate columns excluding ID from OMOP tables in a given dataset'
+        )
+
+        super().__init__(issue_numbers=JIRA_ISSUE_NUMBERS,
+                         description=desc,
+                         affected_datasets=[cdr_consts.UNIONED],
```

Comment:
@nishanthpp93 This needs to be in combined cleaning classes right?

Reply:
You're right, it is in combined.